index.html

```diff
@@ -39,7 +42,10 @@
       property="twitter:description"
```

Comment:
How about changing the `twitter:url` property to `twitter:site` because I don't something like `twiiter:url` is there
For reference :
https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

Real Dev Squad Twitter Handle tag: `@RealDevSquad`